prow/config/jobs/istio.io.yaml

```diff
@@ -14,6 +14,13 @@ jobs:
     command: [entrypoint, prow/integ-suite-kind.sh, test.kube.presubmit]
     requirements: [kind]
     modifiers: [optional]
+  
+  - name: k8s-multicluster-tests
+    command: [entrypoint, prow/integ-suite-kind.sh, doc.test.multicluster]
+    requirements: [kind]
+    env:
+      - name: TOPOLOGY
```

Comment:
I think we generally make new tests optional by default until we decide they are stable.

Reply:
Done.